metaflow/runtime.py

```diff
@@ -198,8 +198,19 @@ def persist_constants(self, task_id=None):
         self._is_cloned[task.path] = task.is_cloned
 
     def execute(self):
-
-        self._logger("Workflow starting (run-id %s):" % self._run_id, system_msg=True)
+        if MFGUI_URL:
+            self._logger(
+                "Workflow starting (run-id %s), see it in the UI at %s"
+                % (
+                    self._run_id,
+                    os.path.join(MFGUI_URL, self._flow.name, self._run_id),
```

Comment:
Should this be `urlib.parse.urljoin` (or a similar function) that does url joins? os.path.join would result in the wrong url on non-unix envs. Not sure how if Metaflow is used on non-envs though.

Reply:
the current assumption is mf only works on linux like systems. Your point remains valid but I think the assumption is pretty pervasive.

Reply:
Note: we switched to just using manual string construction.  `urljoin` does not do what we want (e.g. `urljoin("http://A/B", "C") == "http://A/C"`.